extension/audit-resolv.json

```diff
@@ -0,0 +1,11 @@
+{
+  "812|newman>postman-collection>marked": {
+    "ignore": 1
+  },
+  "812|newman>postman-runtime>postman-collection>marked": {
+    "ignore": 1
+  },
+  "755|nyc>istanbul-reports>handlebars": {
+    "remind": 1555487460723
+  }
+}
```

Comment:
newline

Reply:
it's auto generated by the npm-audit-resolve plugin and I don't want to mess around with it. ^^